lib/ios/RNNButtonBuilder.m

```diff
@@ -0,0 +1,55 @@
+#import "RNNButtonBuilder.h"
+#import "RNNFontAttributesCreator.h"
+
+@interface RNNButtonBuilder ()
+@property(weak, nonatomic) UIViewController<RNNLayoutProtocol> *viewController;
+@property(strong, nonatomic) RNNReactComponentRegistry *componentRegistry;
+@end
+
+@implementation RNNButtonBuilder
+
+- (instancetype)initWithComponentRegistry:(id)componentRegistry {
+    self = [super init];
+    self.componentRegistry = componentRegistry;
+    return self;
+}
+
+- (void)bindViewController:(UIViewController<RNNLayoutProtocol> *)viewController {
```

Comment:
The builder pattern is used in direct substitution to a class' constructor, therefore all parameters the constructor requires should be available. Using a method prefixed with `bind` sounds foreign here.

Let's rename it to `setViewController` and have the method return the builder, this way we can chain its invocation with the `build()` method.